playpen/ansible/roles/dev/files/bashrc

```diff
@@ -82,6 +82,16 @@ pclean() {
     done
     popd
 
+    # Scrub out the Squid cache
+    if [ -d /var/spool/squid ]; then
+        sudo systemctl stop squid
+        sudo rm -rf /var/spool/squid
+        sudo mkdir --context=system_u:object_r:squid_cache_t:s0 --mode=750 /var/spool/squid
```

Comment:
Would it be easier to rm -rf /var/spool/squid/* so you don't need to mkdir it again? Of course, that will only work if regular users can ls that folder so that glob matches. If it doesn't, I bet you can sudo bash -c "rm -rf /var/spool/squid/*". Might make things a little simpler.